cola/libdialect/graphs.cpp

```diff
@@ -110,6 +110,10 @@ LineSegment_SP BoundingBox::buildSideSegment(CardinalDir side) const {
         return std::make_shared<LineSegment>(Point(x, Y), Point(X, Y));
     case CardinalDir::NORTH:
         return std::make_shared<LineSegment>(Point(x, y), Point(X, y));
+    default:
+        COLA_ASSERT(false);
+        return std::make_shared<LineSegment>(Point(0, 0), Point(0, 0));
```

Comment:
Again, if just
```
default:
    COLA_ASSERT(false);
```
alone is enough to silence the warning, then I really prefer not to return an arbitrary and meaningless value.
Will it work without the `return` statement?